packages/playground-examples/copy/ja/JavaScript/External APIs/TypeScript with Deno.ts

```diff
@@ -0,0 +1,32 @@
+//// { order: 3 }
+
+// Denoはセキュリティに重点を置いた開発中のv8ベースの
+// JavaScript及びTypeScriptのランタイムです。
+
+// https://deno.land
+
+// Denoはサンドボックスベースの権限システムを採用しており、JavaScript
+// がファイルシステムやネットワークへのアクセスを減らし、httpベースの
+// インポートを使用してダウンロードしローカルにキャッシュしています。
+
+// 以下はdenoを使用した例をです:
+
+import compose from "https://deno.land/x/denofun/lib/compose.ts";
+
+function greet(name: string) {
+  return `Hello, ${name}!`;
+}
+
+function makeLoud(x: string) {
+  return x.toUpperCase();
+}
+
+const greetLoudly = compose(makeLoud, greet);
+
+// 出力 "HELLO, WORLD!."
```

Comment:
動詞として言葉を補ってあげたほうが日本語としてこなれている気がするので以下でどうでしょう？
```suggestion
// "HELLO, WORLD!."を出力します
```

Reply:
ありがとうございます！修正しました！